backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -1,12 +1,21 @@
 :_content-type: ASSEMBLY
 [id="about-installing-oadp"]
-= About installing OADP
+= Introduction to installing OADP
 include::_attributes/common-attributes.adoc[]
 :context: about-installing-oadp
+:_content-type: CONCEPT
+// as it is not really a prerequisites list
+// should we keep that same pattern
+// to resolve the duplication
 
 toc::[]
 
-As a cluster administrator, you install the OpenShift API for Data Protection (OADP) by installing the OADP Operator. The OADP Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
+As a cluster administrator, you use the OpenShift API for Data Protection (OADP) by installing the OADP Operator. The OADP Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}], and OpenShift plugins for Velero to use, for backup and restore operations.
```

Comment:
i think that in the preview I see velero 1.9
but there might be a comment for that already

Reply:
(I was looking at the wrong place, I see the previews now, sat.)